toonz/sources/toonzlib/preferences.cpp

```diff
@@ -255,6 +255,8 @@ Preferences::Preferences()
     , m_fitToFlipbookEnabled(false)
     , m_previewAlwaysOpenNewFlipEnabled(false)
     , m_autosaveEnabled(false)
+    , m_autosaveSceneEnabled(true)
+    , m_autosaveOtherFilesEnabled(true)
```

Comment:
I can see your point.  I enabled it by default because I would think that most people would want to autosave everything.  I think the number of folks would would only want to save the scene is significantly smaller than those who want to save everything.  I will make the change, but I think there is room for discussion around this.